text_extensions_for_pandas/io/test_conll.py

```diff
@@ -731,6 +732,43 @@ def test_compute_accuracy(self):
             ),
         )
 
+    def test_maybe_download_dataset(self):
+        base_dir = "test_data/io/test_conll"
```

Comment:
Need to replace this fixed location with a temporary directory so that subsequent runs of this test won't see data from previous runs. Also make sure to delete the temp directory if the test passes.

Reply:
fixed in latest commit.